aip_x2_launch/launch/sensing.launch.xml

```diff
@@ -34,6 +34,7 @@
     <include file="$(find-pkg-share vehicle_velocity_converter)/launch/vehicle_velocity_converter.launch.xml">
       <arg name="input_vehicle_velocity_topic" value="/vehicle/status/velocity_status"/>
       <arg name="output_twist_with_covariance" value="/sensing/vehicle_velocity_converter/twist_with_covariance"/>
+      <arg name="config_file" value="$(find-pkg-share individual_params)/config/$(env VEHICLE_ID default)/aip_xx1/vehicle_velocity_converter.param.yaml" />
```

Comment:
`config_path` include `aip_xx1` for all products.
To make it work even if each product doesn't have its own config  file, it might be better to specify `"$(find-pkg-share vehicle_velocity_converter)/config/vehicle_velocity_converter.param.yaml"`. (After the product has prepared the config file in the individual_params, product representatives will make another PR for itself.)

Reply:
You're right, thank you for pointing it out.
Instead of your suggestion, I just removed the "config_file" arguments from x1 & x2 so that the `vehicle_velocity_converter` will load the default parameter designated in `vehicle_velocity_converter.launch.xml`.
https://github.com/tier4/aip_launcher/pull/79/commits/96639f1f388702f39dede7dadd60425fc9b3954c